redis/docker-compose-sentinel-tls.yml

```diff
@@ -0,0 +1,62 @@
+services:
```

Comment:
Does this docker-compose file work?

Reply:
Yes, redis looks like it comes up correctly, it's just connecting over the container networking is difficult, so this file does have limited use in development. I think it's still good to have a working example of sentinel with tls, however.